eng/scripts/CodeCheck.ps1

```diff
@@ -263,7 +263,9 @@ try {
 
         if ($changedAPIBaselines.count -gt 0) {
             LogError ("Detected modification to baseline API files. PublicAPI.Shipped.txt files should only " +
-                "be updated after a major release. See /docs/APIBaselines.md for more information.")
+                "be updated after a major release, and PublicAPI.Unshipped.txt files should not " +
+                "be updated in release branches. If this is an rc branch and you've updated a PublicAPI.Unshipped.txt " +
```

Comment:
But updating in release/x.x-preview is ok...

Reply:
Good call, updated